move.py

```diff
@@ -4,168 +4,137 @@
 from trytond.model import fields
 from trytond.pool import Pool, PoolMeta
 from trytond.pyson import Equal, Eval, Not
-from trytond.transaction import Transaction
-from trytond.modules.product import price_digits
+from trytond.modules.product import price_digits, round_price
 try:
     from trytond.modules.account_invoice_discount import discount_digits
 except ImportError:
     discount_digits = price_digits
 from trytond.modules.currency.fields import Monetary
 
-__all__ = ['Move']
-
 _ZERO = Decimal(0)
 STATES = {
     'invisible': Not(Equal(Eval('state', ''), 'done')),
     }
-PARTIES = {
-    'stock.shipment.in': 'supplier',
-    'stock.shipment.in.return': 'supplier',
-    'stock.shipment.out': 'customer',
-    'stock.shipment.out.return': 'customer',
-    'stock.shipment.internal': 'company',
-    }
 
 
 class Move(metaclass=PoolMeta):
     __name__ = 'stock.move'
-    gross_unit_price = fields.Function(Monetary('Gross Price',
-        digits=price_digits, currency='currency', states=STATES),
-        'get_origin_fields')
-    amount = fields.Function(Monetary('Amount',
-        digits='currency', currency='currency'), 'get_origin_fields')
+
+    base_price = Monetary(
```

Comment:
When will the base price has a value? stock_valued module require extras-depends to sale_discount module and set when base_price when create new moves?